src/styles.scss

```diff
@@ -133,3 +135,49 @@ button {
 .App-right-menu {
   width: 13.75rem;
 }
+
+.ErrorSplash {
+  min-height: 100vh;
+  padding: 20px 0;
+  overflow: auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .ErrorSplash-messageContainer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+
+    padding: 40px;
+    background: #fdf4f5;
+    border: 3px solid #d72b3f;
```

Comment:
is this color from: https://yeun.github.io/open-color/

Reply:
Maybe use the orange tone instead to have it less prominent...?

Reply:
It's a critical error. Should be prominent.